app/controllers/FileChecksController.scala

```diff
@@ -119,16 +121,45 @@ class FileChecksController @Inject() (
         ).uncache()
       } else {
         if (isJudgmentUser) {
-          Ok(views.html.judgment.judgmentFileChecksProgress(consignmentId, reference, applicationConfig.frontEndInfo, request.token.name)).uncache()
+          if (applicationConfig.blockAutomateJudgmentTransfers) {
+            Ok(views.html.judgment.judgmentFileChecksProgress(consignmentId, reference, applicationConfig.frontEndInfo, request.token.name)).uncache()
+          } else {
+            val totalSleepTime = 3 * 60 * 1000 // Total sleep time in milliseconds (5 minutes)
```

Comment:
Might be unnecessary but would it be useful to have these as an environmental variable?

Reply:
Yeah, that makes sense. Create a environment variable.